lib/linkedin_scraper/profile.rb

```diff
@@ -77,7 +77,7 @@ def summary
     end
 
     def picture
-      @picture ||= (@page.at(".profile-picture img").attributes["src"].value.strip if @page.at(".profile-picture img"))
+      @picture ||= (@page.at('.profile-picture img').attributes.values_at('src','data-delayed-url').compact.first.value.strip if @page.at('.profile-picture img'))
```

Comment:
If I am right the problem could only occur with IMG tag with no SRC and no data-delayed-url attribute.

But yes, to be defensive `rescue nil` would be fine.